pkg/virt-operator/resource/generate/components/deployments.go

```diff
@@ -41,10 +41,9 @@ const (
 	VirtAPIName        = "virt-api"
 	VirtControllerName = "virt-controller"
 	VirtOperatorName   = "virt-operator"	
-	PrometheusUrl	   = "prometheus.kubevirt.io"
-	KubeVirtUrl	   = "kubevirt.io"
-	KubernetesUrl	   = "kubernetes.io/hostname"
-	PortCmdLineOption  = "--port"
+	PrometheusAppLabel	   = "prometheus.kubevirt.io"
+	KubeVirtAppLabel	   = "kubevirt.io"
```

Comment:
for this one we have a constant: https://github.com/kubevirt/kubevirt/blob/master/staging/src/kubevirt.io/client-go/api/v1/types.go#L568. I think I did not express this in an understandable fashion.